src/interface/src/app/standalone/planning-areas/planning-areas.component.scss

```diff
@@ -138,32 +138,20 @@ table.planning-areas {
   padding: 20px;
 }
 
-// placeholders, remove once the components are ready.
-
-.search-bar-placeholder {
-  border: 1px solid $color-text-gray;
-  padding: 8px;
-  border-radius: 4px;
-  width: 213px;
+.overlay-loader {
+  position: fixed;
+  inset: 0;
+  background-color: rgba(255, 255, 255, 0.50);
+  z-index: 1000;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
This makes me wonder if we should have a component just for overlays sometime, but not for this PR, of course.

Reply:
yeah thought the same, Im still not 100% convinced if this is going to be the final form, but definitely should be a component if we want this behavior somewhere else